modules/hfs/samples/example.cpp

```diff
@@ -0,0 +1,88 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
In samples the license header can be removed at all.

Reply:
Oh sorry, I seem to misinterpret your comments :)